src/app/shared/components/language-fab/languageFab.component.html

```diff
@@ -1,15 +1,15 @@
 <md-fab-speed-dial class="md-scale md-fab-bottom-right" md-open="false" md-direction="up" ng-cloak>
     <md-fab-trigger>
-        <md-button class="md-fab md-primary md-hue-2">
+        <md-button id="langButton" class="md-fab md-primary md-hue-2">
             <md-tooltip md-delay="400" md-direction="left">
                 <span translate>LANGUAGE_FAB.TOOLTIPS.CHANGE_LANGUAGE</span>
             </md-tooltip>
             <md-icon class="material-icons">language</md-icon>
         </md-button>
     </md-fab-trigger>
     <md-fab-actions>
-        <md-button ng-repeat="language in $ctrl.languages track by $index"
-            ng-click="$ctrl.useLanguage(language)" class="md-fab md-primary md-mini">
+        <md-button ng-repeat="language in $ctrl.languages track by $index" ng-click="$ctrl.useLanguage(language)"
+            class="md-fab md-primary md-mini langSelector">
```

Comment:
css class should be in kebab-case, and since this is a "special" selector for e2e test, name it in a more specific way, like `e2e-langauge-selector`

Reply:
In fact I wasnt using it, deleted it!